### ISSUE TITLE
Could not read path '${RN_PROJECT_DIRECTORY}\node_modules\@babel\parser\.bin\parser'.

### ISSUE THREAD

**Issue by achmadk** [facebook/react-native#22069]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
Run `react-native info` in your terminal and paste its contents here.
```bash
React Native Environment Info:
    System:
      OS: Windows 10
      CPU: x64 Intel(R) Core(TM) i3-2310M CPU @ 2.10GHz
      Memory: 1.43 GB / 7.91 GB
    Binaries:
      Yarn: 1.12.1 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
      npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  3.2.0.0 AI-181.5540.7.32.5056338
```
## Description
I want to build APK release from React Native project. Unfortunately, I got this error.
```
Starting a Gradle Daemon, 1 incompatible and 3 stopped Daemons could not be reused, use --status for details

FAILURE: Build failed with an exception.

* What went wrong:
Failed to capture snapshot of input files for task ':app:bundleReleaseJsAndAssets' property '$1' during up-to-date check.
> Could not read path '${RN_PROJECT_DIRECTORY}\node_modules\@babel\parser\.bin\parser'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

## Reproducible Demo
Create new React Native project with `react-native-cli`, for example:
```
react-native init sampleApp
cd sampleApp/android && gradlew assembleRelease
```

**Comment by bartolkaruza** [comment]
I assume you tried `./gradlew clean` and restarting your computer? I find Gradle's caches can cause strange behavior often not worth debugging without trying the above.

**Comment by achmadk** [comment]
I just execute `gradlew clean` only, not restart the computer, Mr @bartolkaruza .

**Comment by achmadk** [comment]
Not yet @YanmaY .

**Comment by bartolkaruza** [comment]
> ${RN_PROJECT_DIRECTORY}\node_modules\@babel\parser\.bin\parser

This bit looks wrong, but I can't figure out where that path is configured. The variable is not substituted and I think the parser should be at `\node_modules\.bin\parser` instead of inside @babel.  Could you post a log of `./gradlew assembleRelease --info`?

**Comment by 1016482011** [comment]
I solved the problem by revert yarn to version 1.9.4 just now @YanmaY

**Comment by thomanau** [comment]
I also encountered this. Seems to be a problem with newer Yarn versions on Windows.

**Comment by YanmaY** [comment]
version@1.10.1 is OK @achmadk @thomanau

**Comment by achmadk** [comment]
I have downgraded yarn version to 1.10.1 then it works. Thank you @YanmaY .

**Comment by achmadk** [comment]
I think this problem has been fixed at yarn v1.12.3. You can update to recent version or keep installed yarn version to 1.10.1.

**Comment by bartolkaruza** [comment]
@achmadk why did you re-open this issue? Is there something on React Native side that still needs to be changed? If not, you can close this as resolved; people will find your solution (upgrade yarn) through Google regardless of the issue being closed.

**Comment by achmadk** [comment]
sorry for inconvenience @bartolkaruza . I will close this issue.

**Comment by voanhtuan318** [comment]
@achmadk  @tronginc  How to install yarn version 1.10.1

**Comment by achmadk** [comment]
This issue has been solved since yarn release version 1.12.3, @voanhtuan318 . You can either install yarn version 1.12.3 or latest version (1.13.0)

**Comment by anastely** [comment]
I have the same issue when i run **./gradlew assembleRelease**  

> Could not read path node_modules\@babel\parser\.bin\parser'.
